astropy/coordinates/tests/test_sky_coord.py

```diff
@@ -360,8 +360,8 @@ def test_equal():
     ne = sc1 != sc2
     assert np.all(eq == [True, False])
     assert np.all(ne == [False, True])
-    assert (sc1[0] == sc2[0]) == True  # noqa  (numpy True not Python True)
-    assert (sc1[0] != sc2[0]) == False  # noqa
+    assert sc1[0] == sc2[0]  # (numpy True not Python True)
```

Comment:
No need for the comment anymore.

```suggestion
    assert sc1[0] == sc2[0]
```